code/game/turfs/turf.dm

```diff
@@ -307,4 +312,21 @@ var/const/enterloopsanity = 100
 	if(lowertext(message) == "elbereth")
 		to_chat(vandal, "<span class='notice'>You feel much safer.</span>")
 
-	return TRUE
\ No newline at end of file
+	return TRUE
+
+/turf/verb/engrave()
```

Comment:
this absolutely should not be for all turfs. The existing engraving behaviour is for /turf/simulated/floor and turf/simulated/wall only

Reply:
alright. moving it to /turf/simulated alright, or do you want me to put for just in those two and nowhere else?